SwiftWisdom/Core/Controls/VideoPlayer.swift

```diff
@@ -57,6 +57,12 @@ public struct Video {
     }
 }
 
+extension Video {
+    static public func == (lhs: Video, rhs: Video) -> Bool {
+        return (lhs.fileName == rhs.fileName) && (lhs.type == rhs.type) && (lhs.url == rhs.url)
```

Comment:
Since the initializer takes a `Bundle` as an argument, it's theoretically possible(?) maybe(?) that two Videos differ only in their Bundle?  

Not likely, I know, but a good rule of thumb is to test *all* properties for equality when overriding a `==`, take all properties into account when calculating a hash code, etc.